armadillo/src/main/resources/application.yml

```diff
@@ -60,53 +60,3 @@ logging:
 
 storage:
   root-dir: data
-
----
-# 'basic' profile is a configuration without oidc
-# you can only sign in using basicauth 'admin' 'admin'
-
-spring:
-  security:
-    user:
-      password: admin
-  config:
-    activate:
-      on-profile: basic
-
----
-spring:
-  security:
-    user:
-      password: admin
-    oauth2:
-      client:
-        # Uncomment the following section to enable OIDC login:
-        #        registration:
-        #            molgenis:
-        #               client-id: <your client id>
-        #               client-secret: <your client secret>
-        provider:
-          molgenis:
-            issuer-uri: 'https://auth.molgenis.org'
-      resourceserver:
-        jwt:
-          issuer-uri: 'https://auth.molgenis.org'
-        opaquetoken:
-          client-id: 'b396233b-cdb2-449e-ac5c-a0d28b38f791'
-  config:
-    activate:
-      on-profile: development
-
-logging:
```

Comment:
Should the application log by INFO and only DEBUG for development?

Logging is moved as a whole to basic auth yml

Reply:
agreed, we should just document in README how to switch on debug

Reply:
Is is not in the README afaik but in `application.template.yml` IMHO non blocking